### PR TITLE
Install the File Manager feature

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-30668

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by abitskil — DISMISSED** [review]

File: _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md [hunk 1]
```diff
@@ -149,6 +149,13 @@ Ensure the following transfers have been created:
 
 3. Add the following configuration to your project:
 
+{% info_block warningBox "Note" %}
+
+The precedence of the web server's maximum file size configuration is consistently prior to the application configuration and can be conveniently adjusted using `deploy.*.yml`.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md:154]:
```suggestion
The precedence of the web server's maximum file size configuration is consistently prior to the application configuration and can be adjusted using `deploy.*.yml`.
```

File: _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md [hunk 2]
```diff
@@ -149,6 +149,13 @@ Ensure the following transfers have been created:
 
 3. Add the following configuration to your project:
 
+{% info_block warningBox "Note" %}
+
+The precedence of the web server's maximum file size configuration is consistently prior to the application configuration and can be adjusted using `deploy.*.yml`.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md:154]: @romanhavrylko Please rephrase this sentence, difficult to understand meaning

Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md]: @AlexSlawinski, updated, please take a look

**Review by AlexSlawinski — APPROVED** [review]

File: docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: File Manager feature integration
 description: File Manager lets you upload and manager your assets effectively.
-last_updated: Dec 30, 2022
+last_updated: Aug 07, 2023
 template: feature-integration-guide-template
 related:
 - title: File Manager feature walkthrough
```
Comment (VadymSachenko) [inline docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md:7]:
```suggestion
```

File: docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: File Manager feature integration
 description: File Manager lets you upload and manager your assets effectively.
-last_updated: Dec 30, 2022
+last_updated: Aug 07, 2023
 template: feature-integration-guide-template
 related:
```
Comment (VadymSachenko) [inline docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md:6]:
```suggestion
```

File: docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md [hunk 5]
```diff
@@ -1,10 +1,8 @@
 ---
 title: File Manager feature integration
 description: File Manager lets you upload and manager your assets effectively.
-last_updated: Dec 30, 2022
+last_updated: Aug 07, 2023
 template: feature-integration-guide-template
-related:
-- title: File Manager feature walkthrough
   link: docs/scos/dev/feature-walkthroughs/page.version/file-manager-feature-walkthrough.html
```
Comment (VadymSachenko) [inline docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md:6]:
```suggestion
```

File: _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md [hunk 6]
```diff
@@ -11,14 +11,14 @@ Follow the steps below to install the File Manager feature core.
 
 Install the required features:
 
-| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                    |
-|--------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------|
-| Spryker Core | {{site.version}} | [Spryker core feature integration](/docs/pbc/all/miscellaneous/{{site.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
+| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                                           |
+|--------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{page.version}} | [Spryker core feature integration](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md:16]:
```suggestion
| Spryker Core | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
```

File: docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md [hunk 7]
```diff
@@ -1,11 +1,8 @@
 ---
 title: File Manager feature integration
```
Comment (VadymSachenko) [inline docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md:2]:
```suggestion
title: Install the File Manager feature
```

File: docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md [hunk 8]
```diff
@@ -1,11 +1,8 @@
...
 description: File Manager lets you upload and manager your assets effectively.
```
Comment (VadymSachenko) [inline docs/pbc/all/content-management-system/202307.0/base-shop/install-and-upgrade/install-features/install-the-file-manager-feature.md:3]:
```suggestion
description: Learn how to install the File Manager feature that lets you upload and manage your assets effectively.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**